### PR TITLE
Add note to inform incompatibility issues with certain tomcat versions in SSO-OIDC configurations

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
## Purpose

Fixes [wso2/product-is#12078](https://github.com/wso2/product-is/issues/12078)

### CODE REVIEW

File: en/docs/learn/configuring-single-sign-on-oidc.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ You are now ready to try out OpenID Connect SSO with the Pickup Dispatch and Pic
 2. You will be redirected to the login page of WSO2 Identity Server. Log in using your WSO2 Identity Server credentials (admin/admin). Provide the required consent.
 You will be redirected to the Pickup Dispatch application home page.
 
+    !!! note
```
Comment (gomathyk) [inline en/docs/learn/configuring-single-sign-on-oidc.md:40]: Shall we rephrase this as follows and give it as a note under the "Set up" section so as to warn the user in advance? 

"To avoid errors while redirecting to the home page of the Pickup Dispatch application, upgrade the Tomcat server to the latest version."

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/configuring-single-sign-on-oidc.md (hunk 1 above)
Comment (sanjulamadurapperuma) [reply en/docs/learn/configuring-single-sign-on-oidc.md]: Done. Thanks for the suggestion! :smiley:

**Review by gomathyk — APPROVED** [review]
